.build-for-now.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -x
+
+# manually install this one report dependency. Avoid installing all deps.
```

Comment:
nit: First letter caps w/punctuation or no punctuation on all comments in this file.